README.md

```diff
@@ -1587,9 +1584,15 @@ IEEE Trans. Image Process., 2017, pp. 2192-2202. [PDF](https://www.neuromorphic-
 - <a name="Lee19arxiv"></a>Lee, S., Kim, H., Kim, H.J.,  
 *[Edge Detection for Event Cameras using Intra-pixel-area Events](https://bmvc2019.org/wp-content/uploads/papers/1140-paper.pdf)*,  
 British Machine Vision Conf. (BMVC), 2019. [PDF](https://arxiv.org/pdf/1907.07469)
+- <a name="liu19ieeesigproc"></a> S. Liu, B. Rueckauer, E. Ceolini, A. Huber, and T. Delbruck,  
```

Comment:
This one is duplicated in the section "Neuromorphic Systems". Do you want to move it or do you want to have it in two sections, with one of them being a link to the other one?
See the example Bardow et al. CVPR 2016: it has a main entry and other entries linking to the main one.